apps/elewa-group-website/src/app/libs/elements/layout/elewa-hero/elewa-hero.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="root" [style.background-image]="'url(' + backgroundImage + ')'">
+    <div class="container">
+        <nav class="navbar">
```

Comment:
Seperate component / issue

This is not part of the scope of this issue. Please remove